doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -43,34 +45,250 @@ On your computer, one of the following operating systems:
 
 |supported OS|
 
-You also need to install `Git`_ or `Git for Windows`_ (on Linux and Mac, or Windows, respectively).
+You also need the following:
 
-.. _ug_nrf54h20_gs_test_blinky:
+* `Git`_ or `Git for Windows`_ (on Linux and Mac, or Windows, respectively)
+* `curl`
 
-Testing with the Blinky sample
-******************************
+.. _nrf54h20_gs_installing_software:
 
-The nRF54H20 DK comes preprogrammed with the :zephyr:code-sample:`blinky` sample.
+Installing the required software
+********************************
 
-Complete the following steps to test that the DK works correctly:
+To work with the nRF54H20 DK, follow the instructions in the next sections to install the required tools.
 
-1. Connect the USB-C end of the USB-C cable to the **IMCU USB** port the nRF54H20 DK.
-#. Connect the other end of the USB-C cable to your PC.
-#. Move the **POWER** switch to **On** to turn the nRF54H20 DK on.
-   The switch is located in the top left corner of the DK PCB.
+.. _nrf54h20_install_toolchain:
 
-**LED1** will turn on and start to blink.
+Installing the toolchain
+========================
 
-If something does not work as expected, contact Nordic Semiconductor support.
+You can install the toolchain for the customer sampling of the |NCS| by running an installation script.
+Ensure that you additionally have Git and curl installed.
 
-.. _nrf54h20_gs_installing_software:
 
-Installing the required software
-********************************
+.. tabs::
+
+   .. tab:: Windows
+
+      Follow these steps:
+
+      1. Open git bash.
```

Comment:
```suggestion
      1. Open Git Bash.
```

Relates to Wille's comment - capitalizing both seems to be the most common occurrence online